aws_topology/tests/test_cloudformation.py

```diff
@@ -0,0 +1,67 @@
+from stackstate_checks.base.stubs import topology as top
+from .conftest import BaseApiTest
+from mock import patch
+
+
+def dont_send_parked_relations(self):
+    pass
+
+
+class TestCloudFormation(BaseApiTest):
+
+    def get_api(self):
+        return "cloudformation"
+
+    def get_account_id(self):
+        return "731070500579"
+
+    def get_region(self):
+        return 'eu-west-1'
+
+    @patch(
+        "stackstate_checks.aws_topology.aws_topology.AgentProxy.finalize_account_topology",
+        dont_send_parked_relations
+    )
+    def test_process_cloudformation_stack_relations(self):
+        self.check.run()
+        topology = [top.get_snapshot(self.check.check_id)]
+        self.assertEqual(len(topology), 1)
+        self.assert_executed_ok()
+
+        components = topology[0]["components"]
+        relations = topology[0]["relations"]
+
+        stack1 = "arn:aws:cloudformation:eu-west-1:731070500579:stack/stackstate-topo-publisher/" + \
+            "71ea3f80-9919-11e9-a261-0a99a68566c4"
+
+        stack2 = "arn:aws:cloudformation:eu-west-1:731070500579:stack/stackstate-topo-cwevents/" + \
+            "077bd960-9919-11e9-adb7-02135cc8443e"
+
+        self.assert_has_component(
+            components,
+            stack1,
+            "aws.cloudformation",
+            checks={
+                "LastUpdatedTime": "2019-06-27T20:23:43.548Z",
+                "StackName": "stackstate-topo-publisher"
+            }
+        )
+        self.assert_has_component(
+            components,
+            stack2,
+            "aws.cloudformation",
+            checks={
+                "LastUpdatedTime": "2019-06-27T20:20:45.336Z",
+                "StackName": "stackstate-topo-cwevents"
+            }
+        )
+
+        self.assert_has_relation(
+            relations,
+            stack1,
+            stack2,
+            type="child of"
+        )
+
+        self.assertEqual(len(components), self.components_checked)
```

Comment:
I wonder if we should specify an error message here (for all these len tests) that just prints out the actual components so you can use that as a starting point for debugging

Reply:
Was thinking the same thing. Other improvement I was thinking about: check the ids so we cam display the unchecked components/relations